### PR TITLE
refactor: add agflow logo docker poppler

### PR DESCRIPTION
**Pull request by firfircelik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,4 +4,4 @@
 
 ## License
 
-MIT © [AgFlow](https://www.agflow.com)
+MIT © [AgFlow](https://www.agflow.com) <img border="0" alt="AgFlow" src="https://raw.githubusercontent.com/agflow/logos/master/agflow-logo.png" width="140" height="100">
```
Comment (ferhatelmas) [inline README.md]: doesn't look nice

maybe put below line and give more width or less height.

Comment (firfircelik) [reply README.md]: OK, I'm gonna fix all of them. Are we gonna add logo in "danger-plugin-no-eslint-disable " ?

Comment (ferhatelmas) [reply README.md]: yes, why not ?

However, first close open ones, we can do it afterwards.

File: README.md [hunk 2]
```diff
@@ -4,4 +4,7 @@
 
 ## License
 
-MIT © [AgFlow](https://www.agflow.com)
+MIT © [AgFlow](https://www.agflow.com) 
+
+###
```
Comment (ferhatelmas) [inline README.md]: this token shouldn't be needed, right ?

Comment (firfircelik) [reply README.md]: Yes, but just took that for like separator. If its gonna be problem I can remove it instantly

Comment (ferhatelmas) [reply README.md]: not a problem but just extra

**Pull request merged**